server/controllers/network-sheet-parser.js

```diff
@@ -0,0 +1,304 @@
+// var multiparty = require("multiparty");
+// var path = require("path");
+// var demoWorkbooks = require(__dirname + "/demo-workbooks");
+
+const { initWorkbook } = require("./helpers");
+
+var semanticChecker = require(__dirname + "/semantic-checker");
+
+var constants = require(__dirname + "/workbook-constants");
+
+
+// const NETWORK_SHEET_NAMES = ["network", "network_optimized_weights"];
+
+// const isNetworkSheet = (sheetName) => {
+//     return NETWORK_SHEET_NAMES.some(function (network) {
+//         return (sheetName.toLowerCase() === network);
+//     });
+// };
```

Comment:
I get why these lines were just commented out and not outright deleted, but it looks like the next round can remove these entirely once this beta settles in.